Tools/PIDML/qaPidML.cxx

```diff
@@ -321,9 +321,15 @@ struct pidml {
   template <std::size_t i, typename T>
   void pidML(const T& track, const int pdgCodeMC)
   {
-    float pidLogits[3] = {model211.applyModel(track), model2212.applyModel(track), model321.applyModel(track)};
+    float pidLogits[3];
+    if (track.p() <= 0.5) {
```

Comment:
is this in agreement with the training setup?

Reply:
I have added an array with momentum threshold values